src/sentry/monitors/endpoints/base_monitor_details.py

```diff
@@ -112,32 +112,34 @@ def update_monitor(
         if "project" in result and result["project"].id != monitor.project_id:
             raise ParameterValidationError("existing monitors may not be moved between projects")
 
+        old_slug = monitor.slug
+        old_status = monitor.status
+        if params:
```

Comment:
@wedamija if you want to pull this into it's own PR happy to approve this part ASAP

Reply:
Misunderstood the change, ignore